js/src/bqplot-standalone.less

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2015 Bloomberg Finance L.P.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../node_modules/@jupyter-widgets/controls/css/labvariables.css';
```

Comment:
cc @jasongrout 

there might be a more elegant way to do this than including `../node_modules` in the path.

Reply:
It looks like a less plugin supports importing from an npm module: https://github.com/less/less.js/issues/1965, https://github.com/less/less-plugin-npm-import